### ISSUE TITLE
Unable to generate new data in preprocessor 

### ISSUE THREAD

**Issue by KarzoKalori** [j3r3m1/urock_processing#28]
Cannot generate building data and vegetation data simultaneously in URock Preprocessor. The current solution is to first generate data for buildings separately, and then create data for vegetation only in a second round.

**Comment by j3r3m1** [comment]
do you use the menu based preprocessor or the processing one ? If menu based I can not help since I have not developped it. If processing one you might open a new issue in the corresponding repository:
https://github.com/j3r3m1/gen_buildveg_vlayers

**Comment by KarzoKalori** [comment]
I don't really know what the difference is, but it's called preprocessor and I find it under "plugins". Attached is a picture of what it looks like.

![Urock_pre](https://user-images.githubusercontent.com/83512843/202218535-879d7262-996f-4375-bda3-8e03faa16d6d.jpg)

**Comment by j3r3m1** [comment]
Yes this is the menu-based one. Can you please try to download the plugin gen_buildveg_vlayer (https://github.com/j3r3m1/gen_buildveg_vlayers) and install it instead ?
